src/entity/DevCard.ts

```diff
@@ -22,6 +33,15 @@ export class DevCard {
   @Column({ type: 'text', nullable: true })
   background: string | null;
 
+  @Column({ type: 'enum', enum: DevCardTheme, default: DevCardTheme.Default })
```

Comment:
Usually we store enums as string in the DB so we will not have to change the schema